kyuubi-main/src/test/scala/org/apache/kyuubi/operation/WithKyuubiServer.scala

```diff
@@ -72,5 +72,6 @@ trait WithKyuubiServer extends KyuubiFunSuite {
     super.afterAll()
   }
 
-  protected def getJdbcUrl: String = s"jdbc:hive2://${server.connectionUrl}/;"
+  protected def getJdbcUrl: String =
+    s"jdbc:hive2://${server.connectionUrl}/;${KyuubiConf.FRONTEND_BIND_HOST.key}=127.0.0.1"
```

Comment:
I am not sure whether it could work

Reply:
this will work... but OMG... this is a potentical bug..

Reply:
yes, it should be forbid in engine side

Reply:
Let create an Issue ticket for it, and mark it todo here